src/Template.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Jh\Workflow;
+
+/**
+ * @author Michael Woodward <michael@wearejh.com>
+ * @author Aneurin "Anny" Barker Snook <anny@wearejh.com>
+ */
+class Template
+{
+    const TEMPLATE_DIR = __DIR__ . '/../templates/';
+
+    public function cp($template, $dest)
+    {
+        $this->provision(dirname($dest));
+        copy(static::TEMPLATE_DIR . $template, $dest);
+    }
+
+    private function provision($dest, $of = '/')
+    {
+        $path = '';
+        $d = explode('/', ltrim($dest, '/'));
+        foreach ($d as $dd) {
+            $path .= "/{$dd}";
+            if (! is_dir($path)) {
+                mkdir($path);
+            }
+        }
+    }
+
+    public function repcp($template, $dest, array $replace)
```

Comment:
function name doesn't really make sense to me, much prefer longer to be more clear

Reply:
unless `repcp` is a thing like `cp` is